README.md

```diff
@@ -10,6 +10,7 @@ auth:
     url: https://gitlab.com
     #url: https://privategitlab.example.com
 ```
+In gitlab, project -> settings -> repository ->  deploy-tokens
```

Comment:
What does someone do with the tokens?

Reply:
In gitlab project user must create deploy token with the name 'gitlab-deploy-token' and he will be able to log in in  verdaccio with gitlab-ci variables "$CI_DEPLOY_USER"  "$CI_DEPLOY_PASSWORD"  (https://docs.gitlab.com/ee/user/project/deploy_tokens/).